src/app.ts

```diff
@@ -1110,6 +1110,15 @@ function registerSessionAPIRoutes(
       userMetadataStorage
     );
 
+    app.get(
```

Comment:
this route is actually mounted when `FF_ENABLE_SESSION_LOCK_ENDPOINT` is enabled. Are we sure this is the correct behaviour?

Reply:
You're right, this variable is based on `IS_APPBACKENDLI` but it's more correct to create a new one 
https://github.com/pagopa/io-backend/blob/88b17f878f35f6405f74a157b91fe6cb974d5b82/src/config.ts#L868

Reply:
Renamed `FF_ENABLE_SESSION_LOCK_ENDPOINT` to a more generic `FF_ENABLE_SESSION_ENDPOINTS`